app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  has_many :likes
+
+  def increase_user_counter
+    User.find_by(id: user.id).update(post_counter: user.post_counter.nil? ? 1 : ++user.post_counter)
+  end
+
+  def most_recent_comments
+    comments[0, 5]
+  end

```

Comment:
- [ ] to be sure you're getting the correct comments you should sort them first, which will look something like the below.
```
def recent_comments
  comments.order('created_at Desc').limit(5)
end

```